modules/service/data-service.js

```diff
@@ -17,11 +18,15 @@ class DataService {
     }
 
     async initialize() {
-        this.implementation = new GraphDB({
-            repositoryName: this.config.graphDatabase.name,
-            username: this.config.graphDatabase.username,
-            password: this.config.graphDatabase.password,
-        });
+        if (this.config.graphDatabase.implementation === 'Blazegraph') {
```

Comment:
Let's move this into constants and have something like: GRAPH_DATABASE {
'BLAZEGRAPH' : 'Blazegraph'
'GRAPHDB' : 'GraphDb'
}